lib/util/fairplay_utils.js

```diff
@@ -202,6 +202,26 @@ shaka.util.FairPlayUtils = class {
     return FairPlayUtils.initDataTransform(initData, contentId, cert);
   }
 
+  /**
+   * ExpressPlay initDataTransform configuration.
+   *
+   * @param {!Uint8Array} initData
+   * @param {string} initDataType
+   * @param {?shaka.extern.DrmInfo} drmInfo
+   * @export
+   */
+  static expressplayInitDataTransform(initData, initDataType, drmInfo) {
```

Comment:
This looks identical to verimatrixInitDataTransform.  Can we dedup?

Reply:
Yes, I'll do it!

Reply:
Done!